src/convertToObject.js

```diff
@@ -7,6 +7,19 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
+    .split(';')
+    .map((entry) => entry.trim())
+    .filter((entry) => entry)
+    .reduce((cssProperties, entry) => {
+      const [key, value] = entry
+        .split(':')
+        .map((spltEntry) => spltEntry.trim());
+
+      cssProperties[key] = value;
+
+      return cssProperties;
```

Comment:
You don't need to call `map` and `filter` separately here. You can combine them into one `reduce` call by checking if the `entry` is not empty inside the `reduce` function, which would be more efficient.